README.md

```diff
@@ -1,3 +1,6 @@
+[![Continuous Integration](https://github.com/ornicar/scalachess/actions/workflows/ci.yml/badge.svg)](https://github.com/ornicar/scalachess/actions/workflows/ci.yml)
+[![Clean](https://github.com/ornicar/scalachess/actions/workflows/clean.yml/badge.svg)](https://github.com/ornicar/scalachess/actions/workflows/clean.yml)
```

Comment:
I think we don't need the second badge. This one is for some tasks like: cleaning dependencies/artifact ...
First badge is good enough.

Reply:
Yeah I saw just now Thanks!